src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +45,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.attachEventListeners();
```

Comment:
it's not hard to keep track of the data flow if you name the methods appropriately.  Having all the code exposed with no encapsulation is what makes hard the code to understand because there are more things you have to keep in your short term memory. local variables, different paths in the control flow